python/pyproject.toml

```diff
@@ -61,15 +61,17 @@ tests = [
     "pylance>=0.25",
     "requests",
     "datafusion",
+    "scikit-learn"
```

Comment:
Do we really need this? It seems like we use just one function from it. I'd prefer we didn't grab that dependency unless we really need it.